onnxruntime/core/session/inference_session_utils.cc

```diff
@@ -0,0 +1,223 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/session/inference_session_utils.h"
+
+namespace onnxruntime {
+
+//---------------------
+//--- local helpers ---
+//---------------------
+
+//--------------------------------------------
+//--- session options related helpers ---
+//--------------------------------------------
+// Below are some helpers that will be used to set corresponding session option values
+
+static Status
+SetIntraOpNumThreads(SessionOptions& session_options,
+                     int value,
```

Comment:
nit: newline between return type and function name

Reply:
Fixed. thanks.